mqtt-cs/src/main/java/org/apache/rocketmq/mqtt/cs/channel/ChannelManager.java

```diff
@@ -63,4 +63,35 @@ public interface ChannelManager {
      */
     int totalConn();
 
+    /**
+     * Refills the publish receive quota for the given channel.
+     * This method is typically called after a publish acknowledgement or other scenarios that refill the quota.
+     * @param channel The channel for which the quota needs to be refilled.
+     * @return true if the quota was successfully refilled, false otherwise.
+     */
+    boolean publishReceiveRefill(Channel channel);
```

Comment:
extract these flow releated method into another class？

Reply:
OK，i will put them to Session.